src/block-ciphers.rst

```diff
@@ -166,23 +166,23 @@ The most common block cipher in current use is AES.
 
 Contrary to its predecessor DES (which we'll look at in more detail in
 the next chapter), AES was selected through a public, peer-reviewed
-competition following an open call for proposals. This competition
-involved several rounds where all of the contestants were presented,
+competition that arose from an open call for proposals. The competition
+involved several rounds of contestant presentations,
```

Comment:
I gotta think about this more: this makes it sound like the contestants just keep presenting their stuff, but in reality they're presenting, tweaking, other people are vetting the designs, et cetera.